GitUI/ScriptsEngine/IRunScript.cs

```diff
@@ -4,5 +4,14 @@
     public interface IRunScript
     {
         void Execute(int scriptId);
+
+        void Execute(int scriptId, IFileListSource files);
+
+        public interface IFileListSource
+        {
+            List<string> GetFiles();
```

Comment:
Public interfaces should avoid returning mutable types.
```suggestion
            IEnumerable<string> GetFiles();
```

Reply:
Accepted, but changed to `IEnumerable<string>?` (See above)